samples/net/rpl_border_router/README.rst

```diff
@@ -0,0 +1,136 @@
+
+Note that there is no support for running RPL border router in QEMU, as the
+border router requires access to a real radio network technology like
+IEEE 802.15.4 which is not available in QEMU.
+
+Building and Running
+********************
+
+By default, the integrated HTTP server is configured to listen at port 80.
+If you want to modify the HTTP server config options, please check
+the configuration settings in :file:`samples/net/rpl_border_router/src/main.c`
+file and also in the :file:`samples/net/rpl_border_router/src/config.h` file.
+
+This sample code supports both static and dynamic (DHCPv4) IP addresses that
+can be defined in the project configuration file
+:file:`samples/net/rpl_border_router/prj.conf` :
+
+.. code-block:: console
+
+	CONFIG_NET_APP_MY_IPV6_ADDR="2001:db8::1"
+	CONFIG_NET_APP_MY_IPV4_ADDR="192.0.2.1"
+
+Note that the IPv4 address is only used for connection to the integrated web
+server that provides admin web page for management purposes. The web server
+can also be connected using IPv6 address. If you do not have a management
+interface, then IPv4 support can be disabled in configuration file by setting
+:option:`CONFIG_NET_IPV4` to "n". The RPL router uses only IPv6 when routing
+the network traffic.
+
+.. code-block:: console
+
+	$ make pristine
+	$ make BOARD=<your-board>
+
+By default, the RPL border router application enables net shell support and
+provides some useful commands for debugging and viewing the network status.
+
+.. code-block:: console
+
+	shell> br repair
+	[rpl-br/shell] [INF] br_repair: Starting global repair...
+
+The **br repair** command will cause the RPL network to re-configure itself.
+
+.. code-block:: console
+
+	shell> net rpl
+	RPL Configuration
+	=================
+	RPL mode                     : mesh
+	Used objective function      : MRHOF
+	Used routing metric          : none
+	Mode of operation (MOP)      : Storing, no mcast (MOP2)
+	Send probes to nodes         : disabled
+	Max instances                : 1
+	Max DAG / instance           : 2
+	Min hop rank increment       : 256
+	Initial link metric          : 2
+	RPL preference value         : 0
+	DAG grounded by default      : no
+	Default instance id          : 30 (0x1e)
+	Insert Hop-by-hop option     : yes
+	Specify DAG when sending DAO : yes
+	DIO min interval             : 12 (4096 ms)
+	DIO doublings interval       : 8
+	DIO redundancy value         : 10
+	DAO sending timer value      : 4 sec
+	DAO max retransmissions      : 4
+	Node expecting DAO ack       : yes
+	Send DIS periodically        : yes
+	DIS interval                 : 60 sec
+	Default route lifetime unit  : 65535 sec
+	Default route lifetime       : 255
+
+	Runtime status
+	==============
+	Default instance (id 30) : 0xa80081e0 (active)
+	Instance DAGs   :
+	[ 1]* 2001:db8::1 prefix 2001:db8::/64 rank 256/65535 ver 255 flags GJ parent 0x00000000
+
+	No parents found.
+
+The **net rpl** command first prints out static compile time configururation
```

Comment:
move the description before the example (and fix the misspelling of "configuration"